fix: gradient picker size for wp6.1

preda-bogdan opened the pull request:
### Summary
<!-- Please describe the changes you made. -->
Added a min with to keep the gradient picker from collapsing in WP6.1

### Will affect visual aspect of the product
<!-- It includes visual changes? -->
YES

### Screenshots <!-- if applicable -->

### Test instructions
<!-- Describe how this pull request can be tested. -->
1. Check the gradient color picker inside Customizer with WP6.1
2. Check that it maintains the proper width

<!-- Issues that this pull request closes. -->
Closes #3689.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

irinelenache commented:
@preda-bogdan Tested the PR and the issue is not entirely fixed:

The width of the gradient picker is fine but i can't change the color of the gradient or add another color in the range. When i'm trying to do it, there is a scrollbar behaving weirdly at the bottom of the picker
 https://vertis.d.pr/v/99rMfB

preda-bogdan commented:
@irinelenache thank you for reporting this, it seams they also changed other styles inside core. I updated them also.
Can you tell me if the problem is solved now?

irinelenache commented:
@preda-bogdan I checked again and the mentioned issue is fixed, there is still a single problem now:

The color picker in the gradient is opening based on the gradient point's position, and if that position is in the left side, the components will overflow. I made a video to better outline the issue https://vertis.d.pr/v/JdJ9Wa

irinelenache commented:
@preda-bogdan Checked again and now everything works fine, thank you for the fix 🚀
